README.md

```diff
@@ -2,24 +2,57 @@
 
 PSPDFKit - The Leading Mobile PDF Framework for iOS and Android. PSPDFKit 9.2 for iOS needs at least Xcode 11.3.1 or higher and supports iOS 11 ([read more](https://pspdfkit.com/guides/ios/current/announcements/version-support/)). 
 
-## Build Instructions
+## Getting Started Step by Step Guide
 
 > **Note:** Read the [Alloy](#alloy) section before building the module if you're using the Alloy framework.
 
-We recommend using [Titanium](https://github.com/appcelerator/titanium) version 8.3.0.GA, or later. You'll also need to install [CocoaPods](http://cocoapods.org) if don't have it installed already:
+This uses the Appcelerator CLI, not Appcelerator Studio, and assumes Xcode 11.3 is installed as default.
+
+#### Configuring the environment
+
+First, let’s make sure our dependencies are installed.
 
 ```bash
-$ [sudo] npm install -g titanium
-$ [sudo] gem install cocoapods
+# First, make sure we’re using Node 8.
+$ nvm use 8 -g			
+
+# Install Appcelerator CLI
+$ npm install appcelerator -g
+
+# Now install titanium
+$ npm install -g titanium
+
+# Install CocoaPods.
+$ gem install cocoapods
+
+# Run through the basic Appcelerator setup. Requires you to log in.
+$ appc setup
+
+# IMPORTANT: Make sure Xcode’s Command Line Tools are installed
+$ sudo xcode-select —install
+
+# IMPORTANT: Also make sure we’ve accepted the Xcode EULA
+$ sudo xcodebuild -license
+
+# Install the latest appcelerator
+$ appc use 7.1.2
 ```
 
-Now, clone the Appcelerator-iOS project to your local machine:
+You can also run `appc use` to list all available versions for Appcelerator. If some of the following steps fail, please try with an older version of appcelerator, such as 7.0.12: `appc use 7.0.12`.
+
+#### Building the PSPDFKit module
+
+Assuming our workspace is located at `~/dev/`, we can now procceed to build the plugin.
```

Comment:
I'd use a generic path and remove this sentence altogether. Below, we could do something like `cd ~/path/to/workspace`